apache/superset
# fix(install): set SUPERSET_VERSION_RC at the right time

fixes #21082 

### SUMMARY
Fixes problem as described in the issue #21082 

### TESTING INSTRUCTIONS
1. execute command below in a environment which doesn't contain any of the superset env variables.
```
[YOUR_SUPERSET_PATH]/RELEASING/make_tarball.sh [VERSION] [RC] [PGP_KEY_NAME]
```
2. this should run without failures.

### ADDITIONAL INFORMATION
fixes #21082
